docker/pypi/wmagent/wmagent-docker-run.sh

```diff
@@ -72,17 +72,40 @@ ln -s $HOST_MOUNT_DIR/srv/wmagent /data/srv/wmagent
 passwdEntry=$(getent passwd $wmaUser | awk -F : -v wmaHome="/home/$wmaUser" '{print $1 ":" $2 ":" $3 ":" $4 ":" $5 ":" wmaHome ":" $7}')
 groupEntry=$(getent group $wmaGroup)
 
+# @TODO: Create needed unix accounts in container, rather than bind mounting files from the host
 # workaround case where Unix account is not in the local system (e.g. sssd)
 [[ -d $HOST_MOUNT_DIR/admin/etc/ ]] || (mkdir -p $HOST_MOUNT_DIR/admin/etc) || exit $?
+
+# Validation step
+# Delete local docker passwd/group files if uucp is not present (so it can be recreated)
+if [ -f "$HOST_MOUNT_DIR/admin/etc/passwd" ]; then
```

Comment:
Those two nested `if` conditions would have the same effect but would be much more readable if written in the form: 
```
[[ -f "$HOST_MOUNT_DIR/admin/etc/passwd" ]] && ! grep -q uucp "$HOST_MOUNT_DIR/admin/etc/passwd" &&  {
        rm $HOST_MOUNT_DIR/admin/etc/passwd
}
```